internal/cmd/flink/command_iam_binding.go

```diff
@@ -9,7 +9,7 @@ type iamBindingOut struct {
 	Region       string `human:"Region" serialized:"region"`
 	Cloud        string `human:"Cloud" serialized:"cloud"`
```

Comment:
Cloud should come before region, it's "AWS us-west-2", not "us-west-2 AWS"
```suggestion
	Cloud        string `human:"Cloud" serialized:"cloud"`
	Region       string `human:"Region" serialized:"region"`
```

Reply:
Changed!